src/test/java/org/solid/testharness/config/ConfigTest.java

```diff
@@ -19,9 +21,7 @@
 
     @Test
     void getTestSubjectDefault() {
-        assertEquals("https://github.com/solid/conformance-test-harness/testserver",
-                config.getTestSubject().stringValue()
-        );
+        assertEquals(iri("https://github.com/solid/conformance-test-harness/testserver"), config.getTestSubject());
```

Comment:
The URIs in the test never need to change as they are not dependent on anything external. I've moved towards using https://example.org to make this more explicit because I agree this looks risky. I'll add a task to review this.